### PR TITLE
[tempest]Enable more test coverage

### PR DESCRIPTION
**Pull request by gibizer**
Depends-On: https://github.com/openstack-k8s-operators/edpm-ansible/pull/482 (need this fix to get a green run)

### CODE REVIEW

**Comment by gibizer** [comment]
nova-operator tempest job:
* https://review.rdoproject.org/zuul/build/9cefbfba24644944a97146df149c18c4
* ~1h40m
* ~1000 test cases (all the core except cinder, swift is temporary disabled)

openstack-operator todays tempest job:
* https://review.rdoproject.org/zuul/build/9db73f1958f940d6b02e71e319799be7
* ~1h
* 9 test cases (tempest.scenario.test_network_basic_ops)

**Comment by gibizer** [comment]
> ```
> test_server_basic_ops[compute,id-7fff3fb3-91d8-4fd0-bd7d-0204f1f180ba,network,smoke]
> 
> curl: (7) Failed to connect to 169.254.169.254 port 80: Connection refused
> ```

For the metadata request I only see the below nothing in upper layers:

https://review.rdoproject.org/zuul/build/19d343ddd86d4c4d8247e49c84ce84dd/log/controller/ci-framework-data/logs/192.168.122.100/log/containers/neutron/ovn-metadata-agent.log#4649-4751

**Comment by gibizer** [comment]
Same nova metadata failure. So it is not random. I'm deploying locally to see if I can reproduce it.

**Comment by gibizer** [comment]
> Same nova metadata failure. So it is not random. I'm deploying locally to see if I can reproduce it.

On my local deploy metadata works so this is CI specific.

**Comment by gibizer** [comment]
I see this in nova-operator as well  https://github.com/openstack-k8s-operators/nova-operator/pull/585 and the same test was green there last week so something is changed

**Comment by gibizer** [comment]
recheck

**Comment by gibizer** [comment]
Tempest is green now. It takes 2 hours to run the current set. We can finetune the list of test cases to maximize the RoI of CI time spent.

**Comment by gibizer** [comment]
I think we hit a good balance here now.
34 tempest scenario tests. It is green now and the job takes 1h13m. 

Future steps:
* as soon as we have migration support enable those scenario test (this will justify having a multi compute setup)
* enable cinder backend so volume scenario tests can be run as well

**Review by stuggi — APPROVED** [review]
/lgtm

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -1,5 +1,39 @@
 ---
+- job:
+    name: openstack-operator-tempest-multinode
+    parent: podified-multinode-edpm-deployment-crc-3comp
+    dependencies: ["openstack-k8s-operators-content-provider"]
+    vars:
+      cifmw_tempest_tempestconf_profile:
+          overrides:
+            compute.min_compute_nodes: 3
```
Comment (tosky) [inline zuul.d/projects.yaml:9]: I'm late, but is this really needed? python-tempestconf should be able to discover the amount of compute nodes

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+            compute-feature-enabled.vnc_console: true
+            compute-feature-enabled.stable_rescue: true
+            compute_feature_enabled.hostname_fqdn_sanitization: true
+            validation.run_validation: true
```
Comment (tosky) [inline zuul.d/projects.yaml:13]: iirc this has been enabled by default for a while in tempest

**Pull request merged**